pytroll_collectors/s3stalker.py

```diff
@@ -69,7 +91,8 @@ def _reset_last_fetch_from_file_list(files):
 
 def _get_files_since_last_fetch(fs, path):
     files = fs.ls(path, detail=True)
-    files = list(filter((lambda x: x['LastModified'] > DatetimeHolder.last_fetch), files))
+    logger.debug(f"Get files since {get_last_fetch}")
```

Comment:
I think this should be
```suggestion
    logger.debug(f"Get files since {get_last_fetch()}")
```
The original would print `<function get_last_fetch at 0x....>`:

```python
In [1]: def foo(): return 'asd'

In [2]: print(f"{foo}")
<function foo at 0x7ff63872a7a0>

In [3]: print(f"{foo()}")
asd
```